pyproject.toml

```diff
@@ -197,7 +197,7 @@ fix = [
 
 [tool.hatch.envs.notebooks.scripts]
 clean = [
-  "jupyter nbconvert --ClearOutputPreprocessor.enabled=True --ClearMetadataPreprocessor.enabled=True --inplace **/*.ipynb **/**/*.ipynb",
+  "jupyter nbconvert --ClearOutputPreprocessor.enabled=True --ClearMetadataPreprocessor.enabled=True --inplace **/*.ipynb **/internal/*.ipynb **/tracing/*.ipynb **/dolly-pythia-fine-tuned/*.ipynb",
```

Comment:
Had to do this as I couldn't find a negation(!) wildcard (for all *.ipynb except evals/*) that works in both windows and unix at the same time.

Reply:
Is there a shell command that works?

Reply:
This one does `find . -name "*.ipynb" -not -path "./tutorials/evals/*" -exec jupyter nbconvert --ClearOutputPreprocessor.enabled=True --ClearMetadataPreprocessor.enabled=True --inplace {} \;`

Reply:
Looks good.